src/engine/shared/demo.cpp

```diff
@@ -685,6 +693,10 @@ const char *CDemoPlayer::Load(class IStorage *pStorage, class IConsole *pConsole
 	// scan the file for interesting points
 	ScanFile();
 
+	// reset slice markers
+	pConfig->m_ClDemoSliceBegin = -1;
+	pConfig->m_ClDemoSliceEnd = -1;
```

Comment:
It doesn't feel right to pull `CConfig` in here just to use these variables as globals